Formula/fdroidcl.rb

```diff
@@ -0,0 +1,24 @@
+class Fdroidcl < Formula
+  desc "F-Droid desktop client"
+  homepage "https://github.com/mvdan/fdroidcl"
+  url "https://github.com/mvdan/fdroidcl/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "532a8c4c93216cbf13378ff409c06a08d48e8baee6119a50ed43dc0ce9ec7879"
+  license "BSD-3-Clause"
+  head "https://github.com/mvdan/fdroidcl.git", branch: "master"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build"
```

Comment:
I tried adding `*std_go_args(ldflags: "-s -w")` here (I saw it used in other Go formulas), but it doesn't work:
```
==> go build -ldflags=-w
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory - fdroidcl
```

(Here it's only `-w`, but I tried with `-s -w` and `-s` and the result was the same)

Reply:
Let's try it in CI to see the error again.

```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
```